php_phongo.c

```diff
@@ -788,7 +763,8 @@ int phongo_execute_query(mongoc_client_t *client, const char *namespace, zval *z
 		mongoc_cursor_set_max_await_time_ms(cursor, query->max_await_time_ms);
 	}
 
-	if (!phongo_advance_cursor_and_check_for_error(cursor TSRMLS_CC)) {
+	if (!phongo_cursor_advance_and_check_for_error(cursor TSRMLS_CC)) {
+		mongoc_cursor_destroy(cursor);
```

Comment:
It should *not* destroy the cursor, as ``phongo_cursor_advance_and_check_for_error`` already does that.

Reply:
`phongo_cursor_advance_and_check_for_error()` no longer destroys the cursor. That change was made because we don't want to destroy it from within the iterator handlers.